webserver/outcropsketch/settings.py

```diff
@@ -21,6 +22,22 @@
 # Load environment variables from .env file in root of git repo
 load_dotenv(PROJECT_ROOT / '.env')
 
+# Base url to serve media files
+MEDIA_URL = '/media/'
+
+# Path where media is stored
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
```

Comment:
```suggestion
MEDIA_ROOT = BASE_DIR / 'media'
```
pathlib rules os.path.join drools